variables.tf

```diff
@@ -161,7 +171,7 @@ variable "enable_kms_key_rotation" {
 variable "kms_key_enable_default_policy" {
   description = "Specifies whether to enable the default key policy. Defaults to `false`"
```

Comment:
Tiny one, as the default has changed
Defaults to `false` -> Defaults to `true`

Reply:
thank you! fixed in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/2858/commits/570065efe5e960321702f022c969d449c47717f6